README.md

```diff
@@ -16,3 +16,14 @@ $ composer require phlib/mutex
 
 ## Usage
 
+```php
+$mutex = new \Phlib\Mutex\MySQL('my-lock', [
+    'host'     => '127.0.0.1',
+    'username' => 'my-user',
+    'password' => 'my-pass',
+    'dbname'   => 'mydb'
```

Comment:
Locking happens on a connection, so the dbname isn't used. Suggest we drop this connection value.